css/intro.css

```diff
@@ -466,24 +452,25 @@
 	bottom: 0;
 	z-index: 5;
 }
-/*============================================================*/
-.menu_close_btn{
+.menu_close_button{
 	position: absolute;
 	width: 16px;
 	height: 16px;
 	top: 16px;
 	right: 21px;
 	cursor: pointer;
+	background-color: transparent;
```

Comment:
you can reset default button's background-color in reset css file

Reply:
Done.